src/main/resources/log4j.properties

```diff
@@ -6,7 +6,7 @@ log4j.appender.R.File=../logs/catalina.out
 log4j.rootLogger=INFO, R, stdout
 log4j.appender.R.MaxFileSize=2MB
 log4j.logger.org.springframework=INFO
-log4j.logger.com.labsynch.labseer=INFO
+log4j.logger.com.labsynch.labseer=DEBUG
```

Comment:
Default for production deploy should be INFO to reduce impact on performance. It would be great if default for Dev = DEBUG